propagation_comparison/Propagation Comparison (CK - BP).py

```diff
@@ -0,0 +1,640 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Comparison of Propagation Modules (BP - CK)
+
+# 
+# 
+# Numerical simulations of the propagation of charged particles through magnetic fields, solving the equation of motion can be achieved in principle with many different algorithms. There are, however, an increasing number of studies that have found that there are two algorithms, which work in general best for propagating charged particles within a magnetic field. These two algorithms are compared and evaluated.
```

Comment:
Just for completeness, could you give some of the referencesfor the `increasing number of studies' and also the reference for the Boris push algorithm?

Reply:
Thanks, that's a good idea! I'll include these references.